openmedia/player/mixer.py

```diff
@@ -0,0 +1,108 @@
+from pygame import mixer
+from track import Track
+import os
+
+class InvalidVolumeError(Exception):
+    pass
+
+track_list = []
+song_index = 0
```

Comment:
`song_index` is a bit ambiguous. Does something like `curr_song_index` sound better to you?